.github/CONTRIBUTING.md

```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
+1. Make sure you have a [GitHub account](https://github.com/signup/free) with [Two-Factor Authentication](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#two-factor)(2FA) enabled on your account. We require partners to add 2FA protection when contributing to Magento repositories.
 1. Sign the [Magento Contributor Agreement](https://magento.com/content/magento-contributor-agreement).
 1. [Fork](https://help.github.com/articles/fork-a-repo/) the [DevDocs repository](https://github.com/magento/devdocs). Remember to [sync your fork](https://help.github.com/articles/syncing-a-fork/) and update branches as needed.
 1. Review the [DevDocs guidelines](#contribution-guidelines).
 
 ## Contribute docs
 
+The following diagram shows the contribution workflow:
+
 ![Contributing workflow](https://devdocs.magento.com/common/images/contribute-write-submit-pr.png)
 
-1. Create a new branch on your fork. Use a name that best describes the work, or references a GitHub issue number.
+**Tip!** If you are not sure where to start contributing, search for issues with the [`help wanted`](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and [`good first issue`](https://github.com/magento/devdocs/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) labels.
+
+### Create a branch
+
+1. Create a new branch from your fork using a name that best describes the work or references a GitHub issue number.
 1. Edit or create markdown (`.md`) files in your branch.
-1. When ready, push your branch to your fork.
+1. Push your branch to your fork.
 
 ### Create a pull request
 
-Create a PR to the [magento/devdocs repository](https://github.com/magento/devdocs). Fill out as much information as possible and link any GitHub issues.
+1. Create a pull request to the [magento/devdocs](https://github.com/magento/devdocs) repository.
 
-In general, you should use `master` as the base branch when creating a PR. If your contribution is related to a release that is in progress, use a version-specific integration branch, such as `develop`.
+   In general, you should use `master` as the base branch when creating a PR. If your contribution is related to a release that is in progress, use a version-specific integration branch, such as `develop`.
 
-Provide links to devdocs for the files you are changing. This is not required if the list is long.
+1. Complete the pull request template.
 
-If you are updating an example from source code, include a link to the file in the repository.
+   We will close your pull request if you fail to complete the template.
 
-The DevDocs team and Maintainers will review the PR and help with formatting and navigation.
+1. After creating a pull request, a DevDocs staff member or maintainer will review it and may ask you to make revisions.
```

Comment:
Should we add a note that a lack of response in a reasonable amount of time will result in a rejected PR?

Reply:
There are two different names for the same entity in this topic:
1. Magento's tech writer team
2. DevDocs staff

It is not clear whether they are the same or different. We should use one name for one entity to avoid ambiguity.